model/lifecycle.go

```diff
@@ -238,6 +238,61 @@ func MarkVersionCompleted(versionId string, finishTime time.Time, updates *Statu
 	return nil
 }
 
+func SetTaskPriority(t task.Task, priority int64, caller string) error {
+	depTasks, err := task.GetRecursiveDependenciesUp([]task.Task{t}, nil)
+	if err != nil {
+		return errors.Wrap(err, "error getting task dependencies")
+	}
+
+	ids := []string{t.Id}
+	ids = append(ids, t.ExecutionTasks...)
```

Comment:
nit: elsewhere you did `ids := append([]string{t.Id}, t.ExecutionTasks...)`

Reply:
done